README.md

```diff
@@ -38,14 +39,14 @@ vars:
 ```
 
 ## Step 4: Disable models for non-existent sources
-Your Recurly connector may not be syncing all tabes that this package references. This might be because you are excluding those tables. If you are not using those tables, you can disable the corresponding functionality in the package by specifying the variable in your dbt_project.yml. By default, all packages are assumed to be true. You only have to add variables for tables you want to disable, like so:
+Your Recurly connector may not be syncing all tabes that this package references. This might be because you are excluding those tables. If you are not using those tables, you can disable the corresponding functionality in the package by specifying the variable in your `dbt_project.yml`. By default, all packages are assumed to be true. You only have to add variables for tables you want to disable, like so: 
```

Comment:
@fivetran-avinash it looks like you completely deleted this description of Step 4. Can we add it back in.